docs/repositories/issues.md

```diff
@@ -66,34 +69,31 @@ Use the options in the cogwheel menu of each issue to:
     See [how to restore ignored issues](#restoring-ignored-issues).
 
     !!! tip
-        Organization admins can [configure who is allowed to ignore issues](../organizations/roles-and-permissions-for-organizations.md#change-analysis-configuration).
+        Organization admins can [configure who is allowed to ignore issues](../organizations/roles-and-permissions-for-organizations.md#change-analysis-configuration).<!-- TODO CY-6604 Is this behavior unchanged? -->
```

Comment:
Are there any caveats on the existing configurable permissions here?

Reply:
yep, we’re maintaining behaviors related with permissions and Coding Standards